### PR TITLE
add amount restrictions for mainnet and goerli ERC contracts

### PR DESCRIPTION
**Pull request by Westlad**
This PR extends the test code that put restrictions on the amount that can be deposited or withdrawn.
image
It selects the contract addresses to put limits on via the environment variable `ETH_NETWORK` which is the same one used to select the truffle configuration for deployment, so it's transparent to the deployer.  Value restrictions are created in the contract migration.

Test in the normal way, although obviously it's not possible to directly test mainnet and goerli values.  However the symmetry between those and the ganache (`ETH_NETWORK=blockchain1`) case.

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -267,28 +267,64 @@ module.exports = {
...
+          address: '0x499d11E0b6eAC7c0593d8Fb292DCBbF815Fb29Ae',
+          amount: '10000000000000000000000',
+        },
+        {
+          name: 'USDC',
+          address: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
+          amount: '1000000000000',
+        },
+      ],
+      mainnet: [
```
Comment (druiz0992) [inline config/default.js:293]: Why aren't USDT and DAI in the list of restricted tokens?

**Review by druiz0992 — CHANGES_REQUESTED** [review]
minor change request

File: config/default.js (hunk 1 above)
Comment (Westlad) [reply config/default.js:293]: The browser doesn't support them as far as I know.

File: config/default.js [hunk 2]
```diff
@@ -267,28 +267,64 @@ module.exports = {
...
-        amount: '10000000000000000000000',
-      },
-      {
-        name: 'USDC',
-        address: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
-        amount: '1000000000000',
-      },
-    ],
+    tokens: {
+      blockchain1: [
```
[the remark this answers is not in the document]
Comment (Westlad) [reply config/default.js:271]: Yes, if that would be helpful, I'll push an update

File: config/default.js (hunk 1 above)
Comment (Westlad) [reply config/default.js:293]: I can add them though if you like, it just increases the deployment cost a little bit.

**Review by druiz0992 — APPROVED** [review]

**Review by daveroga — APPROVED** [review]

**Pull request merged**